ddl/db_test.go

```diff
@@ -596,9 +596,6 @@ func (s *testDBSuite) TestCancelDropTableAndSchema(c *C) {
 		{true, model.ActionDropSchema, model.JobStateRunning, model.StateDeleteOnly, false},
 	}
 	var checkErr error
-	oldReorgWaitTimeout := ddl.ReorgWaitTimeout
```

Comment:
Why delete these lines rather than add mutex lock?

Reply:
> The `ReorgWaitTimeout` is set in ddl_test.go, we needn't set it again. So I remove it.

Reply:
Got it.